2020/hamming-codes/spanish/sentence_translations.json

```
@@ -497,7 +497,7 @@
  {
   "input": "You might complain that a message which gets messed up by only 2 bit flips is pretty weak, and you would be absolutely right.",
   "model": "nmt",
-  "translatedText": "Podrías quejarte de que un mensaje que se estropea con cambios de sólo 2 bits es bastante débil, y estarías en lo cierto.",
+  "translatedText": "Podrías quejarte de que un mensaje que se estropea con un cambio de sólo 2 bits es bastante débil, y estarías en lo cierto.",
```

In this particular sentence I believe the original version is correct, they are he's talking about flip**s**, so there are multiple `cambios`.

---

You are right, but I still think it sounds weird in spanish, I would change it to something like this:
```suggestion
  "translatedText": "Podrías quejarte de que un mensaje que se estropea sólo con cambios en 2 bits es bastante débil, y estarías en lo cierto.",
```

---

Yeah, seems perfect to me.